.gitignore

```diff
@@ -1 +1,2 @@
+.test-results/
```

Comment:
isn't it for the CI only ?

Reply:
Mh yeah indeed but in the other hand the `--report` is available for everyone. Maybe I should make it hidden too? :thinking:

Reply:
>  the `--report` is available for everyone

:+1: ok to gitignore the dir then ;)